src/components/Game/Quiz/Input/Input.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Input extends Component {
```

Comment:
it can be better if you make this component a function based because it does not have any state

Reply:
resolved